README.md

```diff
@@ -0,0 +1,108 @@
+geiss
```

Comment:
Here (and on some other places below) we should also write `Geiss` if the name of the project starts with a capital letter. I understood your change in line 4 that you want to name the project `Geiss` and not `geiss`. If you want your project to be named `geiss`, I am not sure whether line 4 and 42 have to be changed. We have the proper name (like a trademark) on the one hand and the English rule that sentences start with a capital letter on the other.

???